spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/IcebergArrowColumnVector.java

```diff
@@ -59,6 +59,11 @@ public void close() {
     accessor.close();
   }
 
+  // If a column vector is writable or constant, it should override this method and do nothing.
+  // See more details at SPARK-50235, SPARK-50463 (Fixed in Spark 3.5.4)
+  @Override
+  public void closeIfNotWritable() {}
```

Comment:
will rename to `closeIfFreeable` after RC2 available

Reply:
> If a column vector is writable or constant, it should override this method and do nothing.

we have ConstantColumnVector as well, should we do it vector classes which absolutely requires it ?

Reply:
I checked the codebase, other vectors overwrite `void close() {}`, so they won't be affected